plugins/callback/opentelemetry.py

```diff
@@ -131,7 +139,11 @@ def __init__(self, uuid, name, path, play, action, args):
         if sys.version_info >= (3, 7):
             self.start = time.time_ns()
         else:
-            self.start = _time_ns()
+            # Support versions older than 3.7 with opentelemetry-api > 1.12
+            if OTEL_LIBRARY_TIME_NS_ERROR:
+                self.start = int(time.time() * 1e9)
```

Comment:
This case should not happen IIUC, since python 3.6 is not supported in `opentelemetry-api 1.13`, but better play safe